src/test/java/org/broadinstitute/consent/http/service/EmailServiceTest.java

```diff
@@ -554,4 +557,112 @@ void sendDarExpirationReminderMessage() throws Exception {
         any(),
         any());
   }
+
+  @Test
+  void sendExpirationNoticesTest() throws IOException {
+    User user1 = new User();
+    user1.setUserId(123);
+    user1.setDisplayName("John Doe");
+    user1.setEmail("jd@somewhere");
+
+    User user2 = new User();
+    user2.setUserId(124);
+    user2.setDisplayName("Jane Doe");
+    user2.setEmail("jd@somewhereelse");
+
+    DataAccessRequest dar1 = getMockedDar("DAR-12345", UUID.randomUUID().toString(), user1);
+    DataAccessRequest dar2 = getMockedDar("DAR-12346", UUID.randomUUID().toString(), user2);
+
+    when(userDAO.findUserById(user1.getUserId())).thenReturn(user1);
+    when(userDAO.findUserById(user2.getUserId())).thenReturn(user2);
+    List<DataAccessRequest> dars = List.of(dar1, dar2);
+    when(dataAccessRequestDAO.findAgedDARsByEmailTypeOlderThanInterval(any(), any(), any())).thenReturn(dars);
+    when(templateHelper.getTemplate(EmailType.DAR_EXPIRATION_REMINDER.templateName)).thenReturn(mock());
+    when(templateHelper.getTemplate(EmailType.DAR_EXPIRED.templateName)).thenReturn(mock());
+
+    assertDoesNotThrow(()->service.sendExpirationNotices());
```

Comment:
What is the purpose of this assert here? Any thrown exception will cause the test to fail, so there's no need to use `assertDoesNotThrow()`.
```suggestion
    service.sendExpirationNotices();
```